README.md

```diff
@@ -90,6 +90,16 @@ Find it at [contracts/WalletSimple.sol](contracts/WalletSimple.sol)
 
 A test suite is included through the use of the hardhat framework, providing coverage for methods in the wallet.
 
+On first run:
+1. Create a file called `.env` in the root directory. 
+2. Add the following variable:
+   ```
+   PRIVATE_KEY=<Your private key>
+   ```
+Note: `<your private key>` can be from a wallet like Metamask.
```

Comment:
not sure if I want to add this comment here. This is only needed for deploy script

Reply:
This is also needed to run test, even on a freshly cloned master.

Reply:
not sure why would it be needed to run test and something seems wrong. Hardhat shouldn't need this information to run tests